ch2-git-basics/cloning.qmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Cloning an Existing Repository"
+---
+
+## Introduction
+
+This section will guide you through the process of cloning a GitHub repository. Cloning a repository means making a local copy of a project that exists on a "remote" DevOps server such as Axure DevOps, GitHub or GitLab.
+
+This enables you to make changes and test locally before pushing to be approved and merged with the main branch on GitHub.
```

Comment:
Wonder if this is too much extra info. They may not know about pushing, merging and approving yet. I'd be tempted to bring it back to the very specific basics:

"If you have a repository on Azure DevOps, GitHub or GitLab, you generally can't edit or run your work on those platforms, so if you need to work with the contents of a repo, you'll usually need to make a clone."